config/main.py

```diff
@@ -534,9 +532,9 @@ def interface():
 @click.option('-v', '--verbose', is_flag=True, help="Enable verbose output")
 def shutdown(interface_name, verbose):
     """Shut down interface"""
-    if get_interface_mode() == "ALIAS": 
-        interface_name = interface_alias_to_name(interface_name)
-    """Shut down interface"""
+    if get_interface_mode() == "alias": 
+        interface_name = interface_alias(interface_name)
```

Comment:
If interface_alias() returns None. It tries to execute the "ip link <interface_name> down". This is totally invalid.
Why do this needs to be executed with in-appropriate name.